README.md

```diff
@@ -5,3 +5,18 @@ If you're interested in signing the pledge, please submit a PR against index.htm
 If Github isn't your jam, you can [send us a direct message us on Twitter (our DMs are open)](https://twitter.com/neveragaintech).
 
 If neither of these options are possible, please email neveragaindottech at gmail dot com, but please note that this may delay your signature as we must validate your identity. Please send us some way for us to confirm your identity, such as a LinkedIn profile or a homepage.
+
+## To add your name via Github
+
+1. Click the `_signatures` folder.
+2. Click the "Create new file" button in the upper righthand corner.
+3. Name the file something unique, for example: `first_name_last_name_affiliation.md`. (Don't forget the `.md`.)
+4. Use this template to add your signature:
+```
+---
```

Comment:
We may need to be clearer that the `---` is part of the template. I think it's easier to add a template file like `template.md` in the app root (we can do that, right?) and tell people to copy it and rename / modify.